Fix Land frame name

AModSoul opened the pull request:
Fixes Land frame name on Japanese Showcase

coderanger commented:
Should fix this in the pack. Renaming case only will fail on macOS.

AModSoul commented:
> Should fix this in the pack. Renaming case only will fail on macOS.

I renamed the files to keep the casing consistent with the rest of the lands being capitalized L. The pack is already capitalized and the Land frame doesn't show up because of that right now since the files were lower case. I can switch to all of it being lower case l if you want me to. Just let me know

coderanger commented:
Yes I'm saying you should fix the paths in the frame JS file. Or you need to change the name entirely so it's not just a casing change :)

AModSoul commented:
> Yes I'm saying you should fix the paths in the frame JS file. Or you need to change the name entirely so it's not just a casing change :)

I changed the name because my github desktop wasn't seeing the recase on my anyways so I definitely see what you mean. I also added mask for the margin to make it easier to use without pinlines or with only pinlines.
